### ISSUE TITLE
Erreur 400 (Bad request) lors de l'accès à une page HTTPS

### ISSUE THREAD

**Issue by Kcchouette** [AmauryCarrade/OranjeProxy#14]
Les pages HTTPS semblent ne pas fonctionner avec OranjeProxy (j'ai un retour erreur 400 Bad Request ; The plain HTTP request was sent to HTTPS port) par exemple https://www.eff.org/fr

De même, les pages avec un caractère spécial (accent) ne semble pas passer ( http://fr.wikipedia.org/wiki/Wikip%C3%A9dia:Accueil_principal par exemple)

**Comment by ArthurHoaro** [comment]
Je n'ai pas non plus de problème particulier avec les pages en HTTPS. Ce qui est également bizarre c'est qu'au vu des résultats Google, ton erreur `The plain HTTP request was sent to HTTPS port` semble être une erreur nginx et non Apache...

**Comment by Kcchouette** [comment]
Oui, l'erreur se passe bien sur le site en question, ça ne semble pas être un problème interne à mon serveur… À moins que nginx refuse les connexions HTTPS depuis des certificats auto-signé ? ou depuis Apache ?
